File: src/main/java/com/github/nsc/de/shake/parser/node/ImportNode.java
package com.github.nsc.de.shake.parser.node;

public class ImportNode implements Node {

    public static final String EVERYTHING = "*";

    private final String[] imported;

    public ImportNode(String[] imported) {
        this.imported = imported;
    }

    public String[] getImport() {
        return imported;
    }
}

File: parser/src/commonMain/kotlin/io/github/shakelang/shake/parser/node/ImportNode.kt
package io.github.shakelang.shake.parser.node

import io.github.shakelang.parseutils.characters.position.PositionMap

class ImportNode(map: PositionMap, val import: Array<String>) : Node(map) {

    companion object {
        const val EVERYTHING = "*"
    }

    override fun toJson(): Map<String, *> =
        mapOf(
            "name" to "ImportNode",
            "import" to import
        )
}